elisa/src/org/labkey/elisa/ElisaAssayProvider.java

```diff
@@ -430,4 +434,24 @@ public Collection<StatsService.CurveFitType> getCurveFits()
     {
         return List.of(StatsService.CurveFitType.FOUR_PARAMETER_SIMPLEX, StatsService.CurveFitType.LINEAR);
     }
+
+    @Override
+    protected void moveAssayResults(List<ExpRun> runs, ExpProtocol protocol, Container sourceContainer, Container targetContainer, User user, AssayMoveData assayMoveData)
+    {
+        super.moveAssayResults(runs, protocol, sourceContainer, targetContainer, user, assayMoveData); // assay results

```

Comment:
Should maybe surround with `ensureTransaction` (here and elsewhere).

Reply:
This is called inside AssayProvider.moveRuns, which is called by ExperimentService.moveAssayRuns, which is inside of ensureTransaction

Reply:
Yes, but it could be called from somewhere else and does do more than one update, so other callers will need to know to transact.